guides/common/modules/proc_creating-a-user.adoc

```diff
@@ -4,31 +4,36 @@
 Use this procedure to create a user.
 To use the CLI instead of the {ProjectWebUI}, see the xref:cli-creating-a-user_{context}[].
 
-If a new user is not assigned to an organization, their access is limited.
-To grant systems rights to users, assign them to an organization.
-The next time the user logs on to {Project}, the user's account has the correct system rights.
-
 .Procedure
+// This is the core procedure -- for users who have trouble finding the right button.
+// But I'm keeping it super short so that following it doesn't become a nuisance due to describing steps that users can just see in the web UI.
 . In the {ProjectWebUI}, navigate to *Administer* > *Users*.
 . Click *Create User*.
-. In the *Login* field, enter a username for the user.
-. In the *Firstname* and *Lastname* fields, enter the real first name and last name of the user.
-. In the *Mail* field, enter the user’s email address.
-. In the *Description* field, add a description of the new user.
-. Select a specific language for the user from the *Language* list.
-. Select a timezone for the user from the *Timezone* list.
-+
-By default, {ProjectServer} uses the language and timezone settings of the user’s browser.
-. Set a password for the user:
-.. From the *Authorized by* list, select the source by which the user is authenticated.
-** *INTERNAL*: to enable the user to be managed inside {ProjectServer}.
-** *EXTERNAL*: to configure external authentication as described in {InstallingServerDocURL}Configuring_External_Authentication_{project-context}[Configuring External Authentication] in _{InstallingServerDocTitle}_.
-.. Enter an initial password for the user in the *Password* field and the *Verify* field.
+. Enter the account details for the new user.
 . Click *Submit* to create the user.
 
+// After the procedure, this is the place where users can find more details about the fields that are not that obvious from the web UI.
+// Splitting this from the core procedure supports scannability -- because you can see where the clicking ends and where the explanation begins.
+// Introducing this with "for example" supports maintainability -- because we don't commit ourselves to maintaining a complete list.
+For example, you can specify the following account details:
```

Comment:
```suggestion
.Additional details
```
Maybe something like this?

Reply:
"User account details"?

Reply:
Hmm, I don't like using another `.Heading` for this. I would very much prefer to keep all the web UI bits under a single `.Heading`, otherwise I'm afraid it would hurt scannability. (I can explain further if needed.)

I'll try Lena's suggestion for now and stick to the `For example...` wording.